examples/lock-app/efr32/src/AppTask.cpp

```diff
@@ -50,7 +50,7 @@
 
 #define FACTORY_RESET_TRIGGER_TIMEOUT 3000
 #define FACTORY_RESET_CANCEL_WINDOW_TIMEOUT 3000
-#define APP_TASK_STACK_SIZE (1536)
+#define APP_TASK_STACK_SIZE (4096)
```

Comment:
What is the reason for the growth? What was the straw that broke the camel's back? This is a big increase.

Reply:
Enabling unprovisioned_mdns causes an increased stack usage at launch of the app task for all the necessary configurations for the mdns server.  We are increasing it for the time being until we optimize it.

Reply:
It has to do with a case session now being done in `InitServer();` in the app task for the unprovisionned Broadcast. I looked it up with @mkardous-silabs But he can probably provide more details.